dp-terraform/helm/rhacs-terraform/templates/fleetshard-sync.yaml

```
@@ -136,6 +136,18 @@ spec:
           value: {{ .Values.fleetshardSync.printCentralUpdateDiff | quote }}
         - name: ARGOCD_NAMESPACE
           value: {{ .Values.fleetshardSync.argoCdNamespace | quote }}
+        {{- if (.Values.fleetshardSync.gitops).tenantDefaultAppSourceRepoUrl }}
```

nit: Why brackets are needed?
```suggestion
        {{- if .Values.fleetshardSync.gitops.tenantDefaultAppSourceRepoUrl }}
```